src/mds/MDSMap.cc

```diff
@@ -702,30 +637,40 @@ void MDSMap::encode(bufferlist& bl, uint64_t features) const
   encode(data_pools, bl);
   encode(cas_pool, bl);
 
-  __u16 ev = 15;
+  __u16 ev = 9;
   encode(ev, bl);
   encode(compat, bl);
```

Comment:
compat should belong to ev part (encode a empty compat set here

Reply:
Fixed